messaging/src/main/java/org/axonframework/queryhandling/QueryBus.java

```diff
@@ -147,8 +146,39 @@
      * @param <U>              the incremental response types of the query
      * @return query result containing initial result and incremental updates
      */
+    @Deprecated
```

Comment:
The JavaDoc should reference why it's deprecated. In that line of documentation, it should refer to the method/approach which should be used instead of this function.